packages/wonder-blocks-icon-button/src/util/map-tokens-to-variables.ts

```diff
@@ -0,0 +1,54 @@
+type TokenValue = string | number | boolean;
+
+/**
+ * The type containing the CSS variables that can be overridden to customize the
+ * appearance of a component.
+ *
+ * @example
+ * const tokens: TokensAsVariable<ComponentTokensPrefix> = {
+ *     "--wb-ib-node-box-foreground": "red",
```

Comment:
We can update the example in the docs since we updated the prefixing! 
```suggestion
 *     "--wb-c-node-icon-button--box-foreground": "red",
```